tests/test-path.rs

```diff
@@ -14,9 +14,6 @@ fn test_js_file(file_path: &str) {
             Ok(code) => {
                 ctx.put_args(vec![file_path.clone()]);
                 ctx.eval_module_str(code, &file_path);
-                if let JsValue::Bool(false) = ctx.get_global().get("assertPass") {

```

Comment:
Why delete these?

Reply:
I think I deleted it by mistake. I will put it back.

Reply:
Has this been completed?